app/graphql/mutations/payment_providers/adyen.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mutations
+  module PaymentProviders
+    class Adyen < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      graphql_name 'AddAdyenPaymentProvider'
+      description 'Add or update Adyen payment provider'
+
+      argument :api_key, String, required: true
+      argument :merchant_account, String, required: true
+      argument :live_prefix, String, required: false
+      argument :hmac_key, String, required: false
```

Comment:
GraphQL/ExtractInputType: Consider moving arguments to a new input type

Reply:
This message is a bit "cryptic" :D.

The idea behind this is to define the input arguments in an input type. Here for example, it should be in `Types::PaymentProviders::AdyenInput`. After that, in this mutation, you'll have to replace the arguments with the input type using: 

```ruby
input_object_class Types::PaymentProviders::AdyenInput
```

You can find an example with `Types::Invoices::UpdateInvoiceInput` type, defined in `/app/graphql/types/invoices/update_invoice_input.rb`